tests/unit/executor/test_part_handler.py

```diff
@@ -37,6 +38,15 @@
 # pylint: disable=too-many-lines
 
 
+def setup_module():
+    Features.reset()
+    Features(enable_overlay=True)
+
+
+def teardown_module():
+    Features.reset()
```

Comment:
Part handling should be tested with and without overlay enabled. This will be addressed in CRAFT-1604.